README.md

```diff
@@ -158,6 +158,21 @@ Whether the system is connected to Insights; valid values are `present`
 Whether the system automatically updates the dynamic configuration. It is
 enabled by default.
 
+```yaml
+    rhc_insights:
+      ansible_host: "example-host"
+```
+
+Configures the ansible host name with a custom value for the system record
+in Host Based Inventory (HBI). This host name is used in playbooks by remediations.
+It is `null` by default and will use the system host name if not specified.
+
+Possible values of this variable:
+
+* `null` or an empty value: the ansible host name is not changed.
+* `{state: absent}`: the ansible host name is unset in the insights-client config file and Host Based Inventory (HBI) is updated to use the system host name.
+* any other string value: the ansible host name is changed in Host Based Inventory (HBI).
```

Comment:
This is not true: an empty string value `""` removes the Ansible hostname value from the Inventory. The word “other” suggests that the “empty value” above may refer to an ampty string too, but that’s not true either and yields inconsistent results.

I‘d vote for removing the ambiguous wording “empty value” as `value:` is equivalent to `value: null` in YAML. See the conversation above.

```suggestion
* any non-empty string value: the ansible host name is changed in Host Based Inventory (HBI).
```